### DISCUSSION TITLE
GPUEngine: Kernel: operation not supported

### DISCUSSION THREAD

**Discussion by Chapoly1305** [vosen/ZLUDA#248] [Q&A]
I am trying to run a program which was compiled for CUDA 11.8 with Zluda.Once the program started, it says, 
```
GPU: GPU #0 AMD Radeon RX 6800 XT [ZLUDA] (36x0 cores) Grid(288x128)
GPUEngine: Kernel: operation not supported
```
This error message is not found in previous discussions, nor issues. Is there a way to make Zluda prints more information? So I would also be able to provide further information for a feature request. Thanks!

**Comment by lshqqytiger** [comment]
`AMD_LOG_LEVEL=3` makes HIP SDK to record every HIP function call.
However, it is not enough. The release build of ZLUDA, which does not include debug signatures, returns not implemented status instead of throwing unimplemented error.
You can build ZLUDA without `--release` in order to build for debugging.

**Comment by Chapoly1305** [comment]
Hi @lshqqytiger , thanks for the reply. I followed your advise and compiled a debug version and the log as follow. I am not quite sure if this is helpful. If any feature used but not essential, I could just remove from the program. Do you have an idea? Thanks. 

[run.log](https://github.com/user-attachments/files/16398095/run.log)

**Reply by lshqqytiger** [reply]
Unfortunately, I was not able to find anything helpful.
Do you have the source code of the program `VanitySearch`?
